tests/ui/specs/Scroller/WithEditableSelectItemByPress/ScrollerWithEditableSelectItemByPress-specs.js

```diff
@@ -127,4 +258,180 @@ describe('Editable Scroller', function () {
 			await expectItemWrapperClass('tests_ui_apps_Scroller_WithEditableSelectItemByPress_ScrollerWithEditableSelectItemByPress_noBefore');
 		});
 	});
+
+	describe('pointer', function () {
+		beforeEach(async function () {
+			// Enable Edit Mode with pointer.
+			await ScrollerPage.open('WithEditableSelectItemByPress');
+		});
+
+		it('should remove item with remove button when item is clicked', async function () {
+			// Click on Item 0.
+			const item0 = await $('#item0');
+			await item0.click();
+			// Click on remove button.
+			const removeButton = await $('#removeItem');
+			await removeButton.click();
+
+			// Verify: Item 0 is deleted.
+			await expect(await item0.isExisting()).to.be.false();
+		});
+
+		it('should remove item with remove button when item is focused', async function () {
+			// Focus Item 0 with pointer.
+			const item0 = await $('#item0');
+			await browser.performActions([
+				{
+					type: 'pointer',
+					id: 'mouse',
+					actions: [
+						{type: 'pointerMove', duration: 0, x: 0, y: 0, origin: item0}
+					]
+				}
+			]);
+			// Click on Remove Button.
+			const removeButton = await $('#removeItem');
+			await removeButton.click();
+
+			// Verify: Item 0 is deleted.
+			await expect(await item0.isExisting()).to.be.false();
+		});
+
+		it('should hide item', async function () {
+			// Click on Item 0.
+			const item0 = await $('#item0');
+			await item0.click();
+			// Click on Hide Button.
+			const hideButton = await $('#hideItem');
+			await hideButton.click();
+			const showButton = await $('#showItem');
+			await item0.scrollIntoView();
+			await item0.click();
+
+			// Verify: Show Button is displayed - item is hidden.
+			await expect(await showButton.isDisplayedInViewport()).to.be.true();
+
+			// Click on Show Button.
+			await showButton.click();
+			await item0.scrollIntoView();
+
+			// Verify: Show Button is not displayed.
+			await expect(await showButton.isDisplayedInViewport()).to.be.false();
+		});
+
+		it('should change item position', async function () {
```

Comment:
This test occurs fail on TV ui-test.(browser.perfomActions is not a function) It seems to be a problem that difference of java version. Let's skip this test and test will be checked after TV's java version update.
Leave a comment on the reason for skip.